src/java/picard/analysis/CollectJumpingLibraryMetrics.java

```diff
@@ -61,16 +61,16 @@
 )
 public class CollectJumpingLibraryMetrics extends CommandLineProgram {
     static final String USAGE_SUMMARY = "Collect jumping library metrics. ";
-    static final String USAGE_DETAILS = "This tool collects high-level metrics about the " +
-            "presence of outward-facing (jumping) and inward-facing (non-jumping) read pairs within a SAM or BAM file.<br /><br />" +
+    static final String USAGE_DETAILS = "<p>This tool collects high-level metrics about the " +
+            "presence of outward-facing (jumping) and inward-facing (non-jumping) read pairs within a SAM or BAM file." +
             "For a brief primer on jumping libraries, see <a href='http://gatkforums.broadinstitute.org/discussion/6326/jumping-libraries'>" +
-            "the GATK Dictionary</a>." +
-            "<br /><br />." +
+            "the GATK Dictionary</a></p>." +
+
             "This program gets all data for computation from the first read in each pair in which the mapping quality (MQ) tag " +
             "is set with the mate's mapping quality.  If the MQ tag is not set, then the program assumes that the mate's MQ is " +
-            "greater than or equal to MINIMUM_MAPPING_QUALITY (default value is 0).<br /><br /> "+
-            "All the output metrics files are structured text files."  +
-            "<br /><br />" +
+            "greater than or equal to MINIMUM_MAPPING_QUALITY (default value is 0).</p> "+
+            "<p>All the output metrics files are structured text files.  Note, that although some metrics say percent (PCT) they are actually fractions."  +

```

Comment:
whatever language you end up using for the PCT vs. fraction, I think we should converge on one version and use it for all the tools.